Project.toml

```diff
@@ -38,9 +38,9 @@ Downloads = "1.6"
 ImageMorphology = "0.4"
 JLD2 = "0.4, 0.5"
 KernelAbstractions = "0.9"
+Oceananigans = "0.93.1 - 0.94"
+OrthogonalSphericalShellGrids = "0.1.7"
```

Comment:
does this exist?

Reply:
oh I see.. it wasn't registered

https://github.com/CliMA/OrthogonalSphericalShellGrids.jl/commit/5ad39ae766c292332f316289af2f0ef1c0919e55#commitcomment-148922634

Reply:
now we have to bump to 0.1.8